app/assets/javascripts/setupv2.js

```diff
@@ -158,14 +158,6 @@
     location.href = success_path();
   };
 
-  check_progress = function() {
```

Comment:
Why don't we need this anymore?

Reply:
`check_progress` would make a `GET` request to get the progress of the of the ongoing create and import. It would make this request continuously. 
Instead of polling for the progress we are receiving the progress with `ActionCable`.

Reply:
Ah right, :+1: